datahub-frontend/app/graphql/NamedBar.java

```diff
@@ -3,7 +3,7 @@
 
 @javax.annotation.Generated(
     value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
-    date = "2021-05-03T10:56:06-0700"
+    date = "2021-08-12T10:01:57-0700"
```

Comment:
whats up with this? these looks like spurious changes

Reply:
no i just regenerated the graphql types here. it auto places the time when they were last generated.. you can just look past them